orchestrator/dockerdir/etc/orchestrator/orchestrator.conf.json

```diff
@@ -0,0 +1,50 @@
+{
+  "ApplyMySQLPromotionAfterMasterFailover": false,
+  "BackendDB": "sqlite",
+  "Debug": true,
+  "DetachLostReplicasAfterMasterFailover": true,
+  "DetectClusterAliasQuery": "SELECT CONCAT(SUBSTRING(@@hostname, 1, LENGTH(@@hostname) - 1 - LENGTH(SUBSTRING_INDEX(@@hostname,'-',-2))),'.',SUBSTRING_INDEX(@@report_host,'.',-1))",
+  "DetectInstanceAliasQuery": "SELECT @@hostname",
+  "DiscoverByShowSlaveHosts": false,
+  "FailMasterPromotionIfSQLThreadNotUpToDate": true,
+  "HTTPAdvertise": "http://127.0.0.1:80",
+  "HostnameResolveMethod": "none",
+  "InstancePollSeconds": 5,
+  "ListenAddress": ":3000",
+  "MasterFailoverDetachReplicaMasterHost": true,
+  "MySQLHostnameResolveMethod": "@@report_host",
+  "MySQLTopologyCredentialsConfigFile": "/etc/orchestrator/orc-topology.cnf",
+  "OnFailureDetectionProcesses": [
+    "echo 'Detected {failureType} on {failureCluster}. Affected replicas: {countSlaves}' >> /tmp/recovery.log"
+  ],
+  "PostIntermediateMasterFailoverProcesses": [
+    "echo 'Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Successor: {successorHost}:{successorPort}' >> /tmp/recovery.log"
+  ],
+  "PostMasterFailoverProcesses": [
+    "echo 'Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Promoted: {successorHost}:{successorPort}' >> /tmp/recovery.log"
+  ],
+  "PostUnsuccessfulFailoverProcesses": [],
+  "PreFailoverProcesses": [
+    "echo 'Will recover from {failureType} on {failureCluster}' >> /tmp/recovery.log"
+  ],
+  "ProcessesShellCommand": "sh",
+  "RaftAdvertise": "127.0.0.1",
+  "RaftBind": "0.0.0.0",
+  "RaftDataDir": "/var/lib/orchestrator",
+  "RaftEnabled": true,
+  "RaftNodes": [
+    "127.0.0.1"
+  ],
+  "RecoverIntermediateMasterClusterFilters": [
+    ".*"
+  ],
+  "RecoverMasterClusterFilters": [
+    ".*"
+  ],
+  "RecoveryIgnoreHostnameFilters": [],
+  "RecoveryPeriodBlockSeconds": 300,
+  "RemoveTextFromHostnameDisplay": ":3306",
+  "SQLite3DataFile": "/var/lib/orchestrator/orc.db",
+  "SlaveLagQuery": "SELECT TIMESTAMPDIFF(SECOND,ts,NOW()) as drift FROM sys_operator.heartbeat ORDER BY drift ASC LIMIT 1",
+  "UnseenInstanceForgetHours": 1
+}
```

Comment:
Please add `"StatusEndpoint": "/api/status"`

I'm not sure about 3.2 but on 3.1 it overwrites the web UI URL if we don't specify it.

Reply:
done fbf0b920de0098098e618c3e06d10cc065b5deb9